pkg/cmd/promhelperservice/handlers/instanceconfigs/helpers.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2024 The Cockroach Authors.
+
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package instanceconfigs
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+)
+
+type ResultResponse string
+
+const (
+	// resourceName and version for the API
+	resourceName    = "instance-configs"
+	resourceVersion = "v1"
+	// ClusterIDParam is the url parameter
+	ClusterIDParam = "cluster_id"
+	// defaultInstanceConfigDir is the default instance config directory from where the yml files are created/removed.
+	defaultInstanceConfigDir = "/opt/prom/prometheus/instance-configs"
+	instanceConfigFileExt    = "yml"
+	// instanceConfigEnvVar needs to be set to override the defaultInstanceConfigDir
+	instanceConfigEnvVar = "INSTANCE_CONFIG_DIR"
+
+	// Result responses
```

Comment:
I think I'm missing how these `ResultResponse` are actually being used. Shouldn't the status code be enough?

Reply:
I have added this to give more specific information if needed. e.g. if there are different scenarios of INVALID request. But, I will remove for now as it seems redundant.